random1on1/random1on1bot.py

```diff
@@ -15,32 +23,139 @@ def read_config(location):
 
 class Random1on1Bot(Client):
 
-    def __init__(self, *args, **kwargs):
-        # TODO: Figure out which items to go in constructor: config, token ?
-        super().__init__(*args, **kwargs)
+    def __init__(self, token: str, config: Random1on1BotConfig):
```

Comment:
If you add the algorithm as an initialization input variable here, you can call them in a modular fashion.

Reply:
This is a good point, but I think we might want to make it part of the config (which should be easy enough to do if we have just some config name -> algorithm mapping) 

That way if people fork and run on their own server, they can change the algorithm just by changing the string in the config file.